command/whitelist/show/result.go

```diff
@@ -0,0 +1,28 @@
+package show
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/chain"
+	"github.com/0xPolygon/polygon-edge/helper/common"
+)
+
+type ShowResult struct {
+	GenesisConfig *chain.Chain
```

Comment:
You don't need the whole config here

Reply:
fixed 2b3f36d862a048d87be92830397663041e915d68